database/factories/ModelFactory.php

```diff
@@ -95,3 +96,14 @@
 $factory->defineAs(User::class, 'admin', function () use ($factory) {
     return array_merge($factory->raw(User::class), ['role' => 'admin']);
 });
+
+// Campaign Factory
+$factory->define(Campaign::class, function (Generator $faker) {
+
+    return [
+        'id' => $faker->randomNumber(4),
```

Comment:
decided to use this instead of `$faker->addProvider(new FakerCampaignId($faker));` because those only have 3 campaigns. We can add more but I figured this is more likely of a case to use a random 4 digit number instead when we create new campaigns in Rogue.

Reply:
@katiecrane ooo I'm sure this is possible! But at the same time, maybe I'll just change to `randomNumber()`? I think the number of digits the number is is sorta arbitrary because AFAIK, we don't have any validation/requirements about a campaign id before 4 digits?

Reply:
@chloealee as far as I know that is correct!

Reply:
@katiecrane updated! :)

Reply:
@katiecrane also you're right - this is needed when we seed the `campaigns` table with random data because we probably don't want the seeder to create campaigns with the id under `9000` for when we backfill campaign data from Ashes and in case we want the entire database in our local !